From: Sven-Thorsten Dietrich <sdietrich@suse.de>
Subject: Update smp_send_reschedule_allbutself_cpumask mask parameter

Patch patches.arch/x86_sgi_cpus4096-05-update-send_IPI_mask.patch
changes the mask parameter for send_IPI_mask to a pointer.

Update the RT-introduced version of 
smp_send_reschedule_allbutself_cpumask

Signed-off-by: Sven-Thorsten Dietrich <sdietrich@suse.de>

Index: b/arch/x86/kernel/smp.c
===================================================================
--- a/arch/x86/kernel/smp.c
+++ b/arch/x86/kernel/smp.c
@@ -127,7 +127,7 @@ void smp_send_reschedule_allbutself_cpum
 	cpu_clear(smp_processor_id(), mask);
 	cpus_and(mask, mask, cpu_online_map);
 	if (!cpus_empty(mask))
-		send_IPI_mask(mask, RESCHEDULE_VECTOR);
+		send_IPI_mask(&mask, RESCHEDULE_VECTOR);
 }
 
 /*
